[{"raw":{},"required":["Filename","Size"],"package":"python3-bleak-retry-connector","source":"bleak-retry-connector","version":"3.7.0-1","section":"python","priority":"optional","architecture":"all","depends":["python3-async-timeout | python3-supported-min (>= 3.11)","python3-bleak","python3-bluetooth-adapters","python3-dbus-fast","python3:any"],"installedSize":77,"maintainer":"Debian Python Team <team+python@tracker.debian.org>","description":"Connector for Bleak Clients that handles transient connection failures","homepage":"https://github.com/bluetooth-devices/bleak-retry-connector","filename":"pool/main/b/bleak-retry-connector/python3-bleak-retry-connector_3.7.0-1_all.deb","size":20972,"md5":"ce0ebae41e234df586368def3a36003f","sha256":"20400b1ba662aee0bfb652a94990b3191442468c3fbd82bda364f01b6f144f55","descriptionMd5":"94cf3179d80dc95dc44c0b30cf94cbad"}]